### ISSUE TITLE
Congenital Pulmonary Airway Malformation

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7049]
**Preferred term label:**
Congenital pulmonary airway malformation
**Synonyms** CPAM, ? Congenital cystic adenomatoid malformation

**Definition (free text, please give PubMed ID)** Congenital pulmonary airway malformation (CPAM) - previously known as congenital cystic adenomatoid malformation (CCAM) - is a relatively rare developmental malformation of the lower respiratory tract. It is a hamartomatous, dysplastic developmental abnormality of the lung characterized by abnormal airway patterning during lung branching morphogenesis and is formed by abnormal branching of the immature bronchioles      Neonatology 2016;110:101-115  https://doi.org/10.1159/000440894,  PMID: 28638495

**Parent term (use hpo.jax.org/app)** Congenital cystic adenomatoid malformation of the lung HP:0010959
**Comment -  The term CPAM has been recommended as being preferable to the term congenital cystic adenomatoid malformation, since the lesions are cystic or adenomatoid in only a limited number of cases.  Only three types of CPAMs are distinguished at imaging: large cyst CPAM (type I) and small cyst CPAM (type II), which constitute macrocystic CPAMs; and microcystic or solid type (type III) lesions, which have cysts that are smaller than 5 mm in diameter, with no discernible cystic spaces. 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@JeroenBreckpot, @DominicIliescu - Greetings!  Above is another term request.  It is listed in Orpha, but appears it is being used to replace CCAM. Should it be also be listed as a phenotype? We would appreciate your feedback since it may be a term you've seen in your practice. Thank you in advance for all your help.

**Comment by pnrobinson** [comment]
Explore range of subterms for the new term CPAM

**Comment by saintfrison** [comment]
Somes ideas:
Preferred term label: Congenital pulmonary airway malformation
Synonyms CPAM, Congenital cystic adenomatoid malformation **Stocker type 1, 2 or 3**
Comments : 
**CPAM type 0 et type 4 are not included 
	CPAM Type 0= acinar dysplasia= ORPHA:280827 
	CPAM Type 4 = pleuropulmonary blastoma type 1= ORPHA:280854
CPAM Type 2 can be seen in bronchial atresia sequence
CPAM Type 1 and 2 can be associated with sequestration**

**Comment by pnrobinson** [comment]
Thanks @patt1morgan updated the label and added the subterms as recommended.
